webapp/public/README.md

```diff
@@ -0,0 +1 @@
+This directory will contain all of the images uploaded to the webapp.
```

Comment:
where the images?

Reply:
The images are uploaded by the user. Empty by default.
(every time you use the web app, your image will be uploaded here)